packages/utils/src/severity.ts

```diff
@@ -1,20 +1,28 @@
-import { Severity } from '@sentry/types';
+import { Severity, SeverityLevel } from '@sentry/types';
 
-import { SeverityLevel, SeverityLevels } from './enums';
+// TODO: Should `severityFromString` be deprecated?
+
+// Note: Ideally the `SeverityLevel` type would be derived from `validSeverityLevels`, but that would mean either
+//
+// a) moving `validSeverityLevels` to `@sentry/types`,
+// b) moving the`SeverityLevel` type here, or
+// c) importing `validSeverityLevels` from here into `@sentry/types`.
+//
+// Option A would make `@sentry/types` a runtime dependency of `@sentry/utils` (not good), and options B and C would
+// create a circular dependency between `@sentry/types` and `@sentry/utils` (also not good). So a TODO accompanying the
+// type, reminding anyone who changes it to change this list also, will have to do.
```

Comment:
In light of these options, I agree. I don't think we'd want additional dependencies, much less circular ones...